VK/VK/Presentation/Profile/ViewControllers/PhotoResponse.swift

```diff
@@ -0,0 +1,9 @@
+// PhotoResponse.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Структура ответа ВК
```

Comment:
```suggestion
/// Ответ ВК
```
я тоже тут не везде отмечу, но так же коммент от Жени что и так понятно что это структура можно не подписывать